src/components/Cards/Cards.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Data from "./cardData";
+
+import Card1 from "./Card1";
```

Comment:
Get more creative in naming component or variables, 
Eg Card/ CardItem/ Cardlist, CardRow are sample of names you can use.